packages/cli/templates/webcomponents/igc-ts/grid/grid-editing/files/src/app/__path__/DataGridSharedData.ts

```diff
@@ -9,7 +9,7 @@ export class DataGridSharedData {
     let maleCount: number = 0;
     let femaleCount: number = 0;
     for (let i = 0; i < count; i += 1) {
-      const age: number = Math.round(this.getRandomNumber(20, 40));
+      const age: number = Math.round(this.getSecureRandomNumber(20, 40));
```

Comment:
I'm pretty sure that these about the random numbers in dummy data can safely be marked as false positives.